docs/react.md

```diff
@@ -139,19 +150,7 @@ const { channel } = useChannel({ channelName: "your-derived-channel-name", optio
 });
 ```
 
-Please note that attempts to publish to a derived channel (the one created or retrieved with a filter expression) will fail. In order to send messages to the channel called _"your-derived-channel-name"_ from the example above, you will need to create another channel instance without a filter expression.
-
-```javascript
-const channelName = "your-derived-channel-name";
-const options = { ... };
-const deriveOptions = { filter: 'headers.email == `"rob.pike@domain.com"` || headers.company == `"domain"`' }
-const callback = (message) => { ... };
-
-const { channel: readOnlyChannelInstance } = useChannel({ channelName, options, deriveOptions }, callback);
-const { channel: readWriteChannelInstance } = useChannel({ channelName, options }, callback); // NB! No 'deriveOptions' passed here
-
-readWriteChannelInstance.publish("test-message", { text: "message text" });
-```
+Please note that attempts to publish to a derived channel (the one created or retrieved with a filter expression) using channel instance will fail. Use `publish` function from `useChannel` hook instead.
```

Comment:
```suggestion
Please note that attempts to publish to a derived channel (the one created or retrieved with a filter expression) using channel instance will fail, since derived channels support only `subscribe` capability. Use `publish` function returned by `useChannel` hook instead.
```

Reply:
my intention to the change above is to explain **why** you can't publish to a derived channel - it simply does not support that.

also, is there a page somewhere on ably.com/docs that explains derived channels? I couldn't find one. if there is, it would be beneficial to put that link here, after the `since derived channels support only subscribe capability` message with text like "more about derived channels".

if there is no such page, then we can link at least Capabilities page: https://ably.com/docs/auth/capabilities, with the text "more about channel capabilities".